test/starlark_tests/apple_dynamic_xcframework_import_tests.bzl

```diff
@@ -259,6 +268,27 @@ def apple_dynamic_xcframework_import_test_suite(name):
         tags = [name],
     )
 
+    # Verify imported XCFrameworks with debug symbols provide them as outputs.
```

Comment:
i think analysis tests alone here are insufficient. need to add an archive test of some distinction, though with the caveat described in `generate_xcframework.bzl`

Reply:
done!